lib/bundle.js

```diff
@@ -164,7 +165,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     if (opts.injectConfig)
       ui.log('ok', '`' + output.bundleName + '` added to config bundles.');
 
-    logBuild(path.relative(process.cwd(), fileName), opts);
+    logBuild(path.relative(env.cwd(), fileName), opts);
```

Comment:
Thanks for your feedback, yes using process.chdir(directory) sounds indeed way easier, will do that.